kernel/kernel-api/src/main/java/io/delta/kernel/internal/ScanBuilderImpl.java

```diff
@@ -86,7 +87,8 @@ public Scan build() {
         // TODO: support timestamp type partition columns
         // Timestamp partition columns have complicated semantics related to timezones so block this
         // for now
-        List<String> partitionCols = protocolAndMetadata.get()._2.getPartitionColumns();
+        List<String> partitionCols = VectorUtils.toJavaList(protocolAndMetadata.get()._2
```

Comment:
nit: this might be cleaner formatting?

```java
List<String> partitionCols = VectorUtils.toJavaList(
	protocolAndMetadata.get()._2.getPartitionColumns()
);
```